package.json

```diff
@@ -53,6 +54,7 @@
   },
   "dependencies": {
     "@aws-sdk/credential-providers": "^3.622.0",
+    "@biomejs/biome": "^1.8.3",
```

Comment:
Just wondering, is there any specific reason why biome can't be a dev dependency?

Reply:
That's a mistake @vecerek